### PR TITLE
Mwmentor/example

### PR DESCRIPTION
**Pull request by mwmentor**
This is an update to the ML 2.0 Ref. Arch. Doc. that we are working on presently.  It is referenced as 20210816a.

### CODE REVIEW

**Comment by mwmentor** [comment]
Ready for review.

File: vuepress/docs/refarch/README.md [hunk 1]
```diff
@@ -1,14 +1,362 @@
-# DDD-Outline
-Here is a general outline of the BCs in use for Mojaloop 2.0.
+<p style="color: red; font-weight: bold">>>>>>  gd2md-html alert:  ERRORs: 0; WARNINGs: 0; ALERTS: 5.</p>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:1]: We can remove this. This looks like a warning from the converter.

File: vuepress/docs/refarch/README.md [hunk 2]
```diff
@@ -1,14 +1,362 @@
...
+<ul style="color: red; font-weight: bold"><li>See top comment block for details on ERRORs and WARNINGs. <li>In the converted Markdown or HTML, search for inline alerts that start with >>>>>  gd2md-html alert:  for specific instances that need correction.</ul>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:2]: We can remove this. This looks like a warning from the converter.

File: vuepress/docs/refarch/README.md [hunk 3]
```diff
@@ -1,14 +1,362 @@
...
 
-![BC-Overview](./assets/images/BC-overview.png)
+<p style="color: red; font-weight: bold">Links to alert messages:</p><a href="#gdcalert1">alert1</a>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:4]: We can remove this. This looks like a warning from the converter.

Same with these "alert" lines.

File: vuepress/docs/refarch/README.md [hunk 4]
```diff
@@ -1,14 +1,362 @@
...
+<a href="#gdcalert2">alert2</a>
+<a href="#gdcalert3">alert3</a>
+<a href="#gdcalert4">alert4</a>
+<a href="#gdcalert5">alert5</a>
 
-Some text.
+<p style="color: red; font-weight: bold">>>>>> PLEASE check and correct alert issues and delete this message and the inline alerts.<hr></p>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:10]: We can remove this. This looks like a warning from the converter.

File: vuepress/docs/refarch/README.md [hunk 5]
```diff
@@ -1,14 +1,362 @@
...
+
+![alt_text](./assets/images/DDD_ProblemSpace-1_20210630.png "image_tooltip")
+
+A number of Core Problems (improvements) have been identified by (Business/Developers/Both Business & Developers).  In order to implement the improvements, “internal” development teams will be tasked with developing the required solutions.  Typically the Subdomains thus identified generate significant value for the Mojaloop system, therefore care is taken to ensure that the services they provide are not compromised.  Examples of Core Problem Subdomains include: Participant Lifecycle Management, Settlements, and Scheduling.
+
+### Generic Problems
+
+
+
+<p id="gdcalert2" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image2.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert3">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:99]: To be replaced with the missing image.

File: vuepress/docs/refarch/README.md [hunk 6]
```diff
@@ -1,14 +1,362 @@
...
+
+A number of Generic Problems (improvements) have been identified by (Business/Developers/Both Business & Developers).  In order to implement the improvements, off-the-shelf solutions will be implemented that require no further customization.  They will require integration with the Mojaloop.  Examples of Generic problem subdomains include Authentication, FRMS, and Platform Monitoring.
+
+
+### Supporting Problems
+
+
+### 
+
+<p id="gdcalert3" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image3.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert4">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:113]: To be replaced with the missing image.

File: vuepress/docs/refarch/README.md [hunk 7]
```diff
@@ -1,14 +1,362 @@
...
+
+**[Space for diagram/map]**
+
+
+### New and Unclassified (non-domain)
+
+
+
+<p id="gdcalert4" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image4.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert5">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:133]: To be replaced with the missing image.

File: vuepress/docs/refarch/README.md [hunk 8]
```diff
@@ -1,14 +1,362 @@
...
+
+
+
+## Solution Space (_High level description and the context map)_
+
+The Solution Space defined by DDD-architecture is focused on how the business problems (improvements) identified in the Problem Space are going to be solved.  As a result it necessarily contains more information and technical details than the Problem Space.  It includes elements such as Ubiquitous Language, Bounded Contexts, and Cross-Cutting Concerns.
+
+
+
+<p id="gdcalert5" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image5.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert6">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:150]: To be replaced with the missing image.

File: vuepress/docs/refarch/README.md [hunk 9]
```diff
@@ -1,14 +1,362 @@
...
+
+
+### Bounded Contexts
+
+The following Bounded Contexts have been identified and implemented in Mojaloop:
+
+(This is a high-level description of each Bounded Context.  A more detailed view follows later in this document)
+
+
+<table>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:175]: Table to be converted to markdown --> https://www.markdownguide.org/extended-syntax/#tables

We can try use this tool to convert them --> https://jmalarcon.github.io/markdowntables/

File: vuepress/docs/refarch/README.md [hunk 10]
```diff
@@ -1,14 +1,362 @@
...
+
+
+### Cross cutting concerns
+
+The following Cross cutting concerns have been identified in Mojaloop:
+
+(This is a high-level description of each Bounded Context.  A more detailed view follows later in this document)
+
+
+<table>
```
Comment (mdebarros) [inline vuepress/docs/refarch/README.md:293]: Table to be converted to markdown --> https://www.markdownguide.org/extended-syntax/#tables

We can try use this tool to convert them --> https://jmalarcon.github.io/markdowntables/

**Review by mdebarros — CHANGES_REQUESTED** [review]
A couple of changes.

Let's discuss it tomorrow morning.

**Comment by mwmentor** [comment]
Okay - let's just dump this push - it was just a test to see that I had the process ok.  I see lots of changes - no worries, I will get them done... thanks :-)

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**